modules/virt-vm-migration-tuning.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-live-migration.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vm-migration-tuning_{context}"]
+= VM migration tuning
+
+Live migration requires the use of a shared storage solution that provides `ReadWriteMany` (RWX) access mode. VM disks must be backed by volumes defined with RWX.
+
+{VirtProductName} checks whether a VM can be live-migrated. If a VM can be live-migrated, the `evictionStrategy` is set to `LiveMigrate`.
+//where does the user set the eviction strategy?
+
+You can adjust your cluster-wide live migration limit settings based on the type of workload and migration scenario. This enables you to control how many VMs migrate at the same time, the network bandwidth you want to use for each migration, and how long {VirtProductName} attempts to complete the migration before canceling the process.
+//where does the user set cluster-wide settings?
```

Comment:
Question 7

Reply:
Same as `.Procedure` in `modules/virt-configuring-live-migration-heavy.adoc`